### PR TITLE
FC Ines

### PR DESCRIPTION
**Pull request by inesfcn**

### CODE REVIEW

File: ines/ines.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+let apples = 20;
+let oranges = 30;
+
+console.log("Line 4 is:", apples === oranges ? true : false);
+console.log("Line 5 is:", apples !== oranges ? true : false);
+console.log("Line 6 is:", apples < oranges ? true : false);
```
Comment (ciamac-da) [inline ines/ines.js]: you missed exercise 4! Check whether apples is less than or equal to oranges.
here is the solution!
console.log(apples <= oranges);

File: ines/ines.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+console.log("Line 7 is:", oranges > apples ? true : false);
+
+let mangoes = 5;
+
+console.log("Line 11 is:", mangoes * apples > mangoes + oranges ? true : false);
+console.log("Line 12 is: ", (mangoes === oranges) === apples ? true : false);
```
Comment (ciamac-da) [inline ines/ines.js]: console.log(apples == oranges == mangoes);

File: ines/ines.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+console.log(
+  "Line 13 is:",
+  apples % mangoes === oranges % mangoes ? true : false
+);
+console.log(
+  "Line 14 is",
+  mangoes + apples > oranges - mangoes ? false : true,
+  "because it's false"
+);
```
Comment (ciamac-da) [inline ines/ines.js]: You also have to add another condition and find an operator which will give a result of true. Which is:
console.log((mangoes + apples) >= (oranges - mangoes));

**Review by ciamac-da — CHANGES_REQUESTED** [review]
Hey Ines. Please read my comments that I added to your code and try to fix your code and push again! Thanks

File: ines/ines.js (hunk 2 above)
Comment (inesfcn) [reply ines/ines.js:12]: and no ternary?

Comment (inesfcn) [reply ines/ines.js]: why only ==? and not ===?

**Review by ciamac-da — APPROVED** [review]

**Pull request merged**